### DISCUSSION TITLE
Crashes fixed

### DISCUSSION THREAD

**Discussion by ProcCodeMonitor** [YimMenu/YimMenu#2238] [General]
Hi everyone I've tested the current version of yim that I've compiled and it no longer crashes

**Comment by pierrelasse** [comment]
Maybe take a look at this commit: https://github.com/YimMenu/YimMenu/commit/ba5610af99da7e450d497ad7c9decdad57880fad

**Comment by ghost** [comment]
I'm personally still getting some random crashes but it's a lot less than before for sure. I would make a new issue but I don't get an error at all so I'm not sure what I'd even say but regardless I'm sure I'm not the only one and it'll be fixed soon.
